lib/cli/cordova.js

```diff
@@ -36,6 +36,17 @@ module.exports = function(argv, callback) {
     
     // stop cordova-cli from tracking us!
     cordovaCommand.push("--no-telemetry");
+    
+    // append --save by default to phonegap plugin add 
+    var pluginCommands = ["add", "remove", "rm"];
+    var pluginAlias = ["plugin","plugins"];
+    if(cordovaCommand[0] == "cordova" && 
```

Comment:
You don't need to check `cordovaCommand[0] == "cordova"` because line 27-29 confirms that cordova is the first element.